azan@fahri.nurul.id/files/azan@fahri.nurul.id/HijriCalendarKuwaiti.js

```diff
@@ -0,0 +1,98 @@
+//source : http://www.al-habib.info/islamic-calendar/hijricalendar-kuwaiti.js
+
+function gmod(n, m) {
+	return ((n % m) + m) % m;
+}
+
+function KuwaitiCalendar(adjust) {
+	var today = new Date();
+	if (adjust) {
+		adjustmili = 1000 * 60 * 60 * 24 * adjust;
+		todaymili = today.getTime() + adjustmili;
+		today = new Date(todaymili);
+	}
+	day = today.getDate();
```

Comment:
There's some undeclared variables here. You can fix these using `let day = ...`.

From ~/.xsession_errors:
```
Cjs-Message: JS LOG: Installing settings for azan@fahri.nurul.id
Cjs-Message: JS LOG: Settings successfully installed for azan@fahri.nurul.id
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 14]: assignment to undeclared variable day
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 15]: assignment to undeclared variable month
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 16]: assignment to undeclared variable year
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 17]: assignment to undeclared variable m
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 18]: assignment to undeclared variable y
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 24]: assignment to undeclared variable a
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 25]: assignment to undeclared variable b
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 35]: assignment to undeclared variable jd
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 42]: assignment to undeclared variable bb
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 43]: assignment to undeclared variable cc
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 44]: assignment to undeclared variable dd
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 45]: assignment to undeclared variable ee
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 57]: assignment to undeclared variable wd
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 60]: assignment to undeclared variable iyear
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 61]: assignment to undeclared variable epochastro
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 62]: assignment to undeclared variable epochcivil
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 64]: assignment to undeclared variable shift1
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 66]: assignment to undeclared variable z
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 67]: assignment to undeclared variable cyc
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 69]: assignment to undeclared variable j
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 70]: assignment to undeclared variable iy
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 72]: assignment to undeclared variable im
Cjs-Message: JS WARNING: [/home/jason/.local/share/cinnamon/applets/azan@fahri.nurul.id/HijriCalendarKuwaiti.js 74]: assignment to undeclared variable id
Cjs-Message: JS LOG: Loaded applet azan@fahri.nurul.id in 36 ms
```

Reply:
Thanks for the feedback
Will be fixed soon
It comes from 3rd party lib